.coveragerc

```diff
@@ -11,9 +11,21 @@ omit =
     distributed/utils_test.py
     distributed/deploy/ssh.py
     distributed/_ipython_utils.py
+    distributed/_version.py
```

Comment:
I can spot a bunch of files that don't exist anymore above this line - could you clean them up?

Reply:
absolutely! thanks for spotting this.